src-java/testing/functional-tests/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
 
     testImplementation("org.assertj:assertj-core:3.24.2")
     testImplementation("org.apache.logging.log4j:log4j-api:2.23.0")
+    testImplementation("org.apache.logging.log4j:log4j-core:2.23.0")
```

Comment:
I bet runtime only is enough, but the question is whether it is needed at all. Logs are collected differently, without this dependency.

Reply:
fixed